renderer/skus/BUILD.gn

```diff
@@ -8,15 +8,24 @@ import("//testing/test.gni")
 source_set("browser_tests") {
   testonly = true
 
-  sources = [ "js_skus_browsertest.cc" ]
+  sources = [
```

Comment:
these should be moved into browser/skus now since we're not using the renderer test